.changelog/2315.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_network_peering: Fixes computed values of `altas_gcp_project_id` and `atlas_vpc_name` by obtaining values from associated container
```

Comment:
knit: don't know if we should write the explanation here, i would probably remove "by obtaining...."

Reply:
rephrase slightly to remove implementation detail and focus more on what changed